tools/amGithubNotifier/main.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2019 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/google/go-github/v26/github"
+	"github.com/prometheus/alertmanager/notify/webhook"
+	"github.com/prometheus/alertmanager/template"
+	"golang.org/x/oauth2"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type ghWebhookReceiverConfig struct {
+	authFile string
+	org      string
+	repo     string
+	portNo   string
+	dryRun   bool
+}
+
+type ghWebhookReceiver struct {
+	ghClient *github.Client
+	cfg      ghWebhookReceiverConfig
+}
+
+type ghWebhookHandler struct {
+	client *ghWebhookReceiver
+}
+
+func main() {
+	/*
+		Example `alerts.rules.yml`:
+		```
+		groups:
+		- name: groupname
+		  rules:
+		  - alert: alertname
+		    expr: up == 0
+		    labels:
+		      severity: info
+		      prNum: '{{ $labels.prNum }}'
+		      org: prometheus
+		      repo: prombench
+		    annotations:
+			  description: 'description of the alert'
+		```
+	*/
```

Comment:
```suggestion
	*/
	log.SetFlags(log.Ltime | log.Lshortfile)
```

This is to show the time and filename/line number with each log